packages/camera/camera/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.10.0+2
```

Comment:
if its breaking change then it should be 0.11.0

Reply:
I don't see a version option for that in the update-release-info tool. https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

I can do it manually if thats how its supposed to be but I want to make sure